recipes/dask-ctl/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "dask-ctl" %}
+{% set version = "2021.1.0a0" %}
```

Comment:
This looks like a pre-release version, is this correct? We only upload relases on conda-forge.

Reply:
Could we just add a `skip` for now and then follow [CFEP-05]( https://github.com/conda-forge/cfep/blob/master/cfep-05.md ) on the feedstock? Also somewhat curious whether the same exception used for `black` might apply here.

Reply:
CFEP-05 can be followed here. No need to `skip`.

Reply:
I'm also happy to hold off for a release, pre-releases will be rare with this package. I expect the first full release will be in the next week or so. I was just trying to get out ahead of things.

Reply:
We have now had a full release so I've updated this PR to point to that.